### DISCUSSION TITLE
[nuxt] [request error] globalThis.__buildAssetsURL is not a function

### DISCUSSION THREAD

**Discussion by pxtrez** [nuxt/nuxt#16550] [Questions] [answered]
The console is bombarded with this error after building the page, starting the server and visiting the page. I use ssr. Any ideas?

**Comment by danielroe** [comment]
Would you provide a reproduction? 🙏

**Comment by pxtrez** [comment]
When I remove all image tags, the next problem appears:
`[nitro] Error while generating error response FetchError: 404  (/__nuxt_error?url=/_nuxt/index.623108ec.mjs&statusCode=404&statusMessage=Cannot+find+static+asset+/_nuxt/index.623108ec.mjs&message=Cannot+find+static+asset+/_nuxt/index.623108ec.mjs&description&data)`

I also tried changing the assets folder to public, the problem is still there.

What I'm doing is:
`yarn build`
`node .output/server/index.mjs`

here some config in nuxt.config.ts:
```
{
  build: {
    transpile: ["@headlessui/vue"],
    postcss: {
      postcssOptions: {
        plugins: {
          tailwindcss: {},
          autoprefixer: {},
        },
      },
    },
  },
  modules: ["@nuxtjs/tailwindcss"],
  css: ["~/assets/css/main.css"],
  ssr: true,
}
```

Hope it can somehow help you. If not, please tell me how can I 'reproduce' it better.

**Reply by lukaszg-g** [reply]
I have the same problem @danielroe

**Reply by danielroe** [reply]
This is a helpful guide to how to create reproduction: https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction

**Reply by xuhua1212** [reply]
I have the same problem, Console error:[nuxt] [request error] globalThis.__buildAssetsURL is not a function  @danielroe

**Reply by kirillkuralesin** [reply]
I have the same problem @danielroe

**Reply by TheAlexLichter** [reply]
@xuhua1212 @kirillkuralesin Instead of writing "I have the same problem", you could simply create a minimal reproduction to make it **way more likely** that the bug is fixed ☺️

**Reply by kirillkuralesin** [reply]
@manniL sorry. the problem was in version 3.0.0-rc.6, upgrading to 3.0.0-rc.8 solved the problem

**Reply by pxtrez** [reply]
> @manniL sorry. the problem was in version 3.0.0-rc.6, upgrading to 3.0.0-rc.8 solved the problem

Gotcha! I had to do something wrong while updating to rc. 8. Thanks for your help.

**Reply by safiullahsarhandi** [reply]
i have 3.0.0-rc.11 but problem exists

**Reply by danielroe** [reply]
try rc12

**Comment by pxtrez** [comment, marked as answer]
Update nuxt to 3.0.0-rc.8, thanks @kirillkuralesin.
